source/hic_hal/freescale/kl27z/usb_config.c

```diff
@@ -68,7 +68,7 @@
 //       <i> when the device is fully operational (bMaxPower)
 //   </h>
 #define USBD_CFGDESC_BMATTRIBUTES   0x80
-#define USBD_CFGDESC_BMAXPOWER      0xFA
+#define USBD_CFGDESC_BMAXPOWER      0x96    // Temporal modification to 300mA for microbit
```

Comment:
Probably should make this configurable with an optional `BOARD_USB_BMAXPOWER` macro. The other `usb_config.c` files should be adjusted too.

(We really need to add a docs file that lists all the per-HIC and per-board customization points like weak functions, global variables, and various macros.)

Reply:
Made this change board specific in the .yaml file.